md-rest-api/src/main/java/com/wipro/ats/bdre/md/rest/ext/FlumePropertiesAPI.java

```diff
@@ -43,15 +43,14 @@
 @RequestMapping("/flumeproperties")
 public class FlumePropertiesAPI extends MetadataAPIBase {
     private static final Logger LOGGER = Logger.getLogger(FlumePropertiesAPI.class);
+    private static final String FLUME = "flume";
+    private static final String CHANNEL = "channel";
     @Autowired
     private ProcessDAO processDAO;
-    @Autowired
-    private PropertiesDAO propertiesDAO;
 
     @RequestMapping(value = {"/createjobs"}, method = RequestMethod.POST)
 
-    public
-    @ResponseBody
+    @ResponseBody public
     RestWrapper createJob(@RequestParam Map<String, String> map, Principal principal) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The Cyclomatic Complexity of this method "createJob" is 11 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)